pom.xml

```diff
@@ -145,15 +145,15 @@
         <apache.io.version>2.7</apache.io.version>
         <io.confluent.ksql.version>7.3.0-0</io.confluent.ksql.version>
         <io.confluent.schema-registry.version>${confluent.version.range}</io.confluent.schema-registry.version>
-        <netty-tcnative-version>2.0.46.Final</netty-tcnative-version>
+        <netty-tcnative-version>2.0.51.Final</netty-tcnative-version>
```

Comment:
we'll need to bump these downstream as well, but this is the netty version that works with this version of vertx